pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -1928,10 +1928,13 @@ func TestFailedImportGC(t *testing.T) {
 	tc.Server(0).JobRegistry().(*jobs.Registry).TestingNudgeAdoptionQueue()
 
 	// In the case of the test, the ID of the table that will be cleaned up due
-	// to the failed import will be one higher than the ID of the empty database
+	// to the failed import will be three higher than the ID of the empty database
 	// it was created in.
+	// We increment the id once for the public schema, a second time for the
+	// "MakeSimpleTableDescriptor" call and a third time in
```

Comment:
I don't quite understand why it is three higher and not two? `MakeSimpleTableDescriptor` uses a "fake" ID that is generated via `getNextPlaceholderDescID`. This is not linked to the IDs used in the cluster, or am I misunderstanding something?

Reply:
Sorry this comment is just incorrect - it is two higher, the code reflects that too.